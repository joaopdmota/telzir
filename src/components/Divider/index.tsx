import React, { FunctionComponent } from 'react';
import styled from 'styled-components';

type DividerProps = {
    space: number;
};

const Divider: FunctionComponent<DividerProps> = ({ space }) => (
    <CustomDivider space={space}></CustomDivider>
);

export const CustomDivider = styled.div`
    display: flex;
    width: 100%;
    height: ${(props: DividerProps) =>
        props.space ? `${props.space}px` : '15px'};
`;

export default Divider;
