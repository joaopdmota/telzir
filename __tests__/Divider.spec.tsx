import React from 'react';
import { shallow } from 'enzyme';
import Divider from '../src/app/components/Divider';

it('should renders Divider', () => {
    const wrapper = shallow(<Divider space={50} />);

    expect(wrapper).toMatchSnapshot();
});
