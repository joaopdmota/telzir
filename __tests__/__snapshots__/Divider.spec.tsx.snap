// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders Divider 1`] = `ShallowWrapper {}`;
